src/python/bokstaver/README.md

```diff
@@ -0,0 +1,80 @@
+
+---
+title: Lærerveiledning - Bokstaver
+level: 1
+language: nb
+---
+
+
+# Om oppgaven {.activity}
+
+I denne oppgaven skal vi programmere skilpadder til å konstruere bokstaver og andre geometriske figurer.
+
+
+## Oppgaven passer til: {.check}
+
+ __Fag__: Matematikk, Kunst og håndtverk
+
+ __Anbefalte trinn__: 7.-10. trinn
+
+ __Tema__: Geometri
+
+ __Tidsbruk__: Enkelttime
+
+
+ ## Kompetansemål {.challenge}
+
+ - [ ] __kunst og håndtverk, 2. trinn__: bygge med enkle geometriske grunnformer
+
+ - [ ] __kunst og håndtverk, 2. trinn__: eksperimentere med enkle geometriske former i konstruksjon og som dekorative formelementer
+
+ - [ ] __Matematikk, 2. trinn__: gjennkjenne og beskrive trekk ved enkle to- og tredimensjonale figurer i forbindelse med hjørner, kanter og flater, og sortere og sette navn på figurene etter disse trekkene
+
+ - [ ] __Matematikk, 10. trinn__: bruke koordinater til å avbilde figurer og utforske egenskaper ved geometriske former, med og uten digitale verktøy
+
+
+
+ ## Forslag til læringsmål {.challenge}
+
+ - [ ] Elevene kan bruke matematiske begreper til å beskrive 2 og 3-dimensjonale figuer.
+
+ - [ ] Elevene kan bruke koordinater til å konstruere 2 og 3-dimensjonale figurer.
+
+
+ ## Forslag til vurderingskriterier {.challenge}
+
+ - [ ] Eleven oppnår middels måloppnåelse ved å fullføre oppgaven.
+
+ - [ ] Eleven oppnår høy måloppnåelse ved å videreutvikle egen kode basert på oppgaven. 
+
+ 
+ ## Forutsetninger og utstyr {.challenge}
+
+ - [ ]  __Forutsetninger__: Ingen
+
+ - [ ]  __Utstyr__: Datamaskin med internett
+
+
+
+
+
+
+
+ # Fremgangsmåte
+
+Her kommer tips, erfaring og utfordringer til de ulike stegene i den faktiske oppgaven. [Klikk her for å se oppgaveteksten.](../bokstaver/bokstaver.html){target=_blank}
+
```

Comment:
I [Skilpadder](http://oppgaver.kidsakoder.no/python/skilpadder/skilpadder.html) finner man kommandoene for velge farge og fylle lukkede figurer med farge, men de er ikke nevnt i denne oppgaven. Dersom elevene trenger hjelp så kan det nevnes at kommandoene er de følgende:
- fillcolor('red')
- pencolor('red')
- begin_fill()
- end_fill()